### ISSUE TITLE
AWSCognitoIdentityService.m - FABKitProtocol.h file not found

### ISSUE THREAD

**Issue by iwllyu** [aws/aws-sdk-ios#351]
Hey I have cocoapods 1.0.0.beta.6, and it installed 

  - AWSCognito (2.3.6):
    - AWSCore (= 2.3.6)
  - AWSCore (2.3.6)

but Xcode is complaining that AWSCognitoIdentityService.m line 30 `#import "FABKitProtocol.h"` Could not be found.

I'm honestly a little lost - In the Pods project under Pods/AWSCore/ I see both AWSCognitoIdentityService.m and FABKitProtocol.h. So what's going on here?

See image for error
http://i.stack.imgur.com/XBH2r.png

And the file obviously existing...

http://i.stack.imgur.com/IdeCh.png

**Comment by orxelm** [comment]
Same issue here, went back to '2.3.4'

**Comment by iwllyu** [comment]
I'm back on 2.3.5 with no issues. I did have to revert my cocoapods to 0.39.0 to fix a missing bundle issue with another lib

**Comment by yosuke-matsuda** [comment]
We do not support beta versions of CocoaPods. Please use the latest non-beta version of CocoaPods. Thanks.

**Comment by iwllyu** [comment]
Sorry I've been really busy with work, I'll try this as soon as I can

**Comment by iwllyu** [comment]
No problems upgrading to 2.3.6 with cocoapods 0.39.0, thanks!

**Comment by dmorrow** [comment]
CocoaPods has released 1.0.0 and this is still an issue

**Comment by orxelm** [comment]
@dmorrow It just released 3 days ago, allow some time for migration.

**Comment by yosuke-matsuda** [comment]
We are not able to reproduce the issue with the AWS Mobile SDK for iOS 2.4.1 and CocoaPods 1.0.0 (GA). Can you share the detailed steps to reproduce the issue? Thanks.

**Comment by dmorrow** [comment]
@yosuke-matsuda I'm not able to reproduce this any longer either.  I think it may have been a conflict with Branch or Segment-GoogleAnalytics.  This is my current Podfile and everything is ok with 2.4.1.

I've noticed issues with CP 1.0.0 that seems to depend which order you pod install certain pods.

```
source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/contentful/CocoaPodsSpecs'
source 'https://github.com/ooyala/ios-sdk-podspecs'

use_frameworks!
inhibit_all_warnings!

workspace '***'
project ***/***App.xcodeproj'

def contentful_pods
    pod 'ContentfulDeliveryAPI'
    pod 'ContentfulPersistence'
end

target :***App do
    platform :ios, '9.0'
    pod 'GoogleAds-IMA-iOS-SDK', '3.0.beta.16' #3.1.0 doesn't work with OoyalaIMASDK
    pod 'Alamofire', '~>3'
    pod 'AlamofireImage', '~>2'
    pod 'Analytics'
    pod 'Segment-Branch'
    pod 'Branch', '~>0.11.18'
    pod 'Segment-GoogleAnalytics', :git => 'https://github.com/rokgregoric/analytics-ios-integration-google-analytics.git'
    pod 'Segment-Mixpanel'
    pod 'Fabric', '~> 1.6'
    pod 'Crashlytics', '~> 3.6'
    contentful_pods
    pod 'Segment-Bugsnag'
    pod 'OoyalaSDK', '~>4'
    pod 'Reveal-iOS-SDK', :configurations => ['Debug']
    pod 'FBSDKLoginKit', '~>4.9'
    pod 'AWSCognito', '~>2'
    pod 'ObjectMapper', '~> 1.1'
    pod 'GoogleConversionTracking', '~> 3.4'
end

target :ContentfulSeedDatabase do
    platform :osx, '10.10'
    contentful_pods
end

# this noops the method at runtime
Pod::Installer.class_eval { def verify_no_static_framework_transitive_dependencies; end }

post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
            if config.build_settings['PRODUCT_NAME'] == "Segment_GoogleAnalytics"
                config.build_settings['LIBRARY_SEARCH_PATHS'] = ["$(inherited)", "$(PODS_ROOT)/GoogleAnalytics/Libraries", "$(PODS_ROOT)/GoogleIDFASupport/Libraries"]
                config.build_settings['OTHER_LDFLAGS'] = %Q{-weak_framework "CoreData" -weak_framework "SystemConfiguration" -l"z" -l"sqlite3" -l"sqlite3.0" -l"GoogleAnalytics"}
            end
        end
    end
end
```

**Comment by orxelm** [comment]
@yosuke-matsuda I can confirm that everything is working properly with the AWS Mobile SDK for iOS 2.4.1 and CocoaPods 1.0. Thanks!

**Comment by zhihuitang** [comment]
same here in Xcode8 and AWSCore (2.3.6), I can see the file "FABKitProtocol.h". I am using cocoapod 1.1.0.rc.3. Anyone can help? thanks a lot


        Using AWSAPIGateway (2.3.6)
        Using AWSCognito (2.3.6)
        Using AWSCore (2.3.6)
        Using AWSMobileAnalytics (2.3.6)

**Comment by iwllyu** [comment]
@zhihuitang can you upgrade your AWS version? 

I upgraded cocoapods to 1.1.1 since cocoapods doesn't work out of the box with 0.39.0 [cocoapod issue](https://github.com/CocoaPods/CocoaPods/issues/6167), [details about sharding](http://blog.cocoapods.org/Sharding/). Got the `FABKitProtocol.h` not found issue. Upgraded AWS to 2.4.12, and the problem was resolved.

**Comment by zhihuitang** [comment]
@iwllyu  I fixed this problem by going back to 2.3.4, I will try cocoapods1.1.1 tomorrow.thanks

    pod 'AWSCognito', '2.3.4'
    pod 'AWSAPIGateway', '2.3.4'
    pod 'AWSMobileAnalytics', '2.3.4'
